src/core/ComplexFilter/style.ts

```diff
@@ -14,57 +15,56 @@ export const Wrapper = styled("div")`
 `;
 
 export const StyledPopper = styled(Popper)`
-  ${fontHeaderXs}
+  & {
+    ${fontHeaderXs}
 
-  .MuiAutocomplete-popperDisablePortal {
-    position: relative;
-    width: 100% !important;
-  }
+    .${autocompleteClasses.popperDisablePortal} {
+      position: relative !important;
+      transform: none !important;
```

Comment:
is `important` necessary here, or can we get away with `&&`?

Reply:
Thanks so much for the quick review, @mmmmmmaya 🙏 !

Just tested it out! I think since `Popper` applies the `transform` as inline style, it seems that `!important` is the only way to fight it 😭 I just added a comment to document this too!

Reply:
@bethbertozzi for fixing Autocomplete dropdown misalignment, `transform: none !important;` was the key! I think Dropdown and MenuSelect could probably reuse most if not all fixes in this PR 😆 

Thank you!

Reply:
thanks @tihuan!